jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -23,18 +23,18 @@ terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.11"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.12"
 terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:0.0.9"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.13"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:0.0.1"
+terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:0.0.1"
```

Comment:
Wondering if this will break the `sed` script in the `Jenkinsfile`:
```
 sed -i "s/$image.*/$image:$version\\"/" jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh
```
Would this mistakenly catch both `terra_jupyter_aou` and `terra_jupyter_aou_old` lines?

The sed script could probably be more robust..

Reply:
hmm didn't realize there's a sed in Jenkinsfile...we should probably hold on this since aou is not sure r major version bump is desirable for them yet

Reply:
I didn't update regex, but instead made sed conditional in jenkinsfile (basically if it's aou, we don't do version replacement)